src/core/src/op/avg_pool.cpp

```diff
@@ -135,31 +60,12 @@ AvgPool::AvgPool(const Output<Node>& arg,
                  const Shape& pads_end,
                  const Shape& kernel,
                  bool exclude_pad,
-                 op::RoundingType rounding_type,
-                 const PadType& auto_pad)
-    : Op({arg}),
-      m_kernel(kernel),
-      m_strides(strides),
-      m_pads_begin(pads_begin),
-      m_pads_end(pads_end),
-      m_exclude_pad(exclude_pad),
-      m_auto_pad(auto_pad),
-      m_rounding_type(rounding_type) {
+                 RoundingType rounding_type,
+                 PadType auto_pad)
```

Comment:
```suggestion
                 const PadType& auto_pad)
```
Use same parameters to avoid change API.